targets/lantiq-xrx200

```diff
@@ -32,7 +32,14 @@ device('avm-fritz-box-3370-rev-2-micron-nand', 'avm_fritz3370-rev2-micron', {
 
 device('avm-fritz-box-7360-sl', 'avm_fritz7360sl', {
 	factory = false,
-	aliases = {'avm-fritz-box-7360-v1', 'avm-fritz-box-7360-v2'},
+	aliases = {'avm-fritz-box-7360-v1'},
+})
+
+device('avm-fritz-box-7360-v2', 'avm_fritz7360-v2', {
+	factory = false,
+	manifest_aliases = {
+		'avm-fritz-box-7360-sl' -- upgrade from OpenWrt 22.03
```

Comment:
The comment seems wrong. OpenWRT 22.03 correctly identifies the device as `avm,fritz7360-v2`.

Reply:
you can read the marker as:
this alias can be dropped once we drop support for Gluon-Releases based on OpenWrt 22.03